lib/nrf9160_time/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NRF9160_TIME
+	bool "nRF9160 time library"
+	select AT_CMD
+	select SNTP
+
+if NRF9160_TIME
+
+config NRF9160_TIME_UPDATE_INTERVAL
+	int "Date time update interval"
+	default 3600
+
+config NRF9160_TIME_MODEM
+	bool "Get date time from the nRF9160 onboard modem"
+	default y if BOARD_NRF9160_PCA10090NS || BOARD_NRF9160_PCA20035NS
+
+config NRF9160_TIME_THREAD_SIZE
+	int "Stack size of the thread maintaining date time"
+	default 2048
```

Comment:
This is quite large, has it been measured?

Reply:
Nope, this I will do.